powerfulseal/policy/action_pods.py

```diff
@@ -154,3 +172,27 @@ def action_check_pod_state(self, pods, params):
                 self.logger.error("Expected pod in state '%s', got '%s' (%r)", state, pod.state, pod)
                 success = False
         return success
+
+    def action_stop_host(self, pods, params):
+        """ Action to stop a node.
+        """
+        self.inventory.sync()
+        host_ips = list(set([p.host_ip for p in pods]))
+        for host_ip in host_ips:
+            host = self.inventory.get_node_by_ip(host_ip)
+            if host is None:
+                self.logger.warning("Couldn't find host with IP %r", host_ip)
+                return False
```

Comment:
so we do not continue once running into a bad ip?

Reply:
That's the idea.